common/src/main/java/com/alibaba/nacos/common/utils/ByteUtils.java

```diff
@@ -17,12 +17,13 @@
 package com.alibaba.nacos.common.utils;
 
 import java.nio.charset.Charset;
-import java.nio.charset.StandardCharsets;
+
```

Comment:
Why  remove constant value？I think use constant value will be better

Reply:
JDK6 does not exist with this constant